### PR TITLE
added `A` binding

### PR DESCRIPTION
**Pull request by TAImatem**

### CODE REVIEW

File: OWML.ModHelper.Menus/ModPopupInput.cs [hunk 1]
```diff
@@ -47,7 +53,7 @@ protected void Subscribe(IModButton button)
 
         protected virtual void OnOpenCommand(SingleAxisCommand command)
         {
-            if (IsSelected && (command == InputLibrary.select || command == InputLibrary.enter || command == InputLibrary.enter2))
+            if (IsSelected && (command == InputLibrary.menuConfirm || command == InputLibrary.enter || command == InputLibrary.enter2))
```
Comment (amazingalek) [inline OWML.ModHelper.Menus/ModPopupInput.cs:56]: `if (IsSelected && _commands.Contains(command))`

Comment (amazingalek) [reply OWML.ModHelper.Menus/ModPopupInput.cs:56]: something went wrong with conflicts i guess

File: OWML.ModHelper.Menus/ModPopupInput.cs [hunk 2]
```diff
@@ -34,9 +42,7 @@ private void SetupCommands()
         {
             var listenerObject = new GameObject();
             CommandListener = listenerObject.AddComponent<ModCommandListener>();
-            CommandListener.AddToListener(InputLibrary.select);
-            CommandListener.AddToListener(InputLibrary.enter);
-            CommandListener.AddToListener(InputLibrary.enter2);
+            _commands.ForEach(command => CommandListener.AddToListener(command));
```
Comment (amazingalek) [inline OWML.ModHelper.Menus/ModPopupInput.cs]: Neither VS nor ReSharper suggested it, but this actually works: `_commands.ForEach(CommandListener.AddToListener);`

Comment (TAImatem) [reply OWML.ModHelper.Menus/ModPopupInput.cs]: done

**Pull request merged**